borg/archiver.py

```diff
@@ -60,10 +59,9 @@ def print_warning(self, msg, *args):
         self.exit_code = EXIT_WARNING  # we do not terminate here, so it is a warning
         logger.warning(msg)
 
-    def print_verbose(self, msg, *args):
-        if self.verbose:
-            msg = args and msg % args or msg
-            logger.info(msg)
+    def print_file_status(self, status, path):
+        if self.args.changed:
+            print("%1s %s" % (status, remove_surrogates(path)), file=sys.stderr)
```

Comment:
this follows the convention that was established in the earlier convention of switching to the logging module. this is the same as what is done with `--stats` and `--progress`, for example.

i believe that if we want to log this on top of displaying it on the terminal, we need to go back to the drawing board and consider logging filters, as i originally suggested.